src/analyses/flow_insensitive_analysis.cpp

```diff
@@ -49,8 +49,7 @@ exprt flow_insensitive_abstract_domain_baset::get_return_lhs(locationt to) const
   // must be the function call
   assert(to->is_function_call());
 
-  const code_function_callt &code=
-    to_code_function_call(to_code(to->code));
+  const code_function_callt &code = to_code_function_call(to->code);
 
   return code.lhs();
```

Comment:
Wishlist item: the `assert` should be removed and this entire code be collapsed to `return to_code_function_call(to->code);`